src/window.rs

```diff
@@ -1127,6 +1127,26 @@ impl Window {
         self.window.request_user_attention(request_type)
     }
 
+    /// Request a frame throttling hint from the windowing system.
+    ///
+    /// The error is returned when frame couldn't be requested and the user should use their
+    /// fallback renderer scheduling logic, like using timer based on monitor refresh rate or
+    /// graphics API extensions (VSYNC).
+    ///
+    /// For more see [`FrameThrottled`] event.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Wayland:**  Uses frame callbacks. The user must perform drawing operation resulting in
+    ///                 `wl_surface.commit`(eglSwapBuffers, etc) after issueing a request.
```

Comment:
@daxpedda does web needs a drawing operation or something like that afterwards like Wayland? Maybe we should state web specifics as well here, like what is used and how it's done?

Reply:
> does web needs a drawing operation or something like that afterwards like Wayland?

No, nothing needs to be done for Web. If you don't draw everything will continue like before, the current buffer won't be invalidated or anything like that.

> Maybe we should state web specifics as well here, like what is used and how it's done?

I think I would prefer to even remove it for Wayland unless there is something that **needs** to be known by the user. "how it's done" should remain an implementation detail. We haven't done it anywhere else as well unless really necessary.

Reply:
The thing is that the callback won't be requested if the user doesn't commit, commit is done implicitly by swap buffers, vulkan stuff, or softbuffer. I'm just trying to communicate that commit is not done by the winit itself, because doing so could cause glitches.

Reply:
That does sounds like something the user needs to know indeed.

By the way, this isn't what I understood from that comment. To me it reads like you must perform a commit, when you request the frame, but it's not specified what happens if you don't. But from what you just wrote, I am understanding that if you don't commit you won't get a `FrameThrottled` event.

```suggestion
    /// - **Wayland:**  `wl_surface.commit` must be called for this event to fire.
    ///                 This is usually done by drawing operations, e.g. `eglSwapBuffers`.
```
Something like that? I have no clue about the terminology btw.

Reply:
If you don't `commit` you won't get a callback. It'll just be in a pending state, because the surface states are applied atomically on commit.